doc/source/getting_started/troubleshooting.rst

```diff
@@ -0,0 +1,42 @@
+.. _troubleshooting:
+
+===============
+Troubleshooting
+===============
+
+
+No huggingface repo access
+==========================
+
+Sometimes, you may face errors accessing huggingface models, such as the following message when accessing `llama2`:
+
+.. code-block:: text
+
+   Cannot access gated repo for url https://huggingface.co/api/models/meta-llama/Llama-2-7b-hf.
+   Repo model meta-llama/Llama-2-7b-hf is gated. You must be authenticated to access it.
+
+This typically indicates either a lack of access rights to the repository or missing huggingface access tokens. 
+The following sections provide guidance on addressing these issues.
+
+Get access to the huggingface repo
+----------------------------------
+
+To obtain access, navigate to the desired huggingface repository and agree to its terms and conditions. 
+As an illustration, for the `llama2` model, you can use this link:
+ `https://huggingface.co/meta-llama/Llama-2-7b-hf <https://huggingface.co/meta-llama/Llama-2-7b-hf>`_.
+
+Set up credentials to access huggingface
+----------------------------------------
+
+Your credential to access huggingface can be found online at `https://huggingface.co/settings/tokens <https://huggingface.co/settings/tokens>`_.
+
+If you are running photons locally, you can do one of the following:
+
+- set the token as an environmental variable, with ``export HUGGING_FACE_HUB_TOKEN=your_token_here``.
+- or, in your python environment, run the following command and login. Huggingface will store the credential
```

Comment:
This may not work since the auth is needed on the server side.

Reply:
Do you mean the second method? Maybe I should delete the second approach and keep the first one.